bundles/org.openhab.binding.visualcrossing/README.md

```diff
@@ -0,0 +1,227 @@
+| humidity        | system.atmospheric-humidity | R          |             |
+| dew             | temperature-channel         | R          |             |
+| precip          | precip-channel              | R          |             |
+| precip-prob     | precip-prob-channel         | R          |             |
+| precip-type     | precip-type-channel         | R          |             |
+| snow            | snow-channel                | R          |             |
+| snow-depth      | snow-channel                | R          |             |
+| wind-gust       | system.wind-speed           | R          |             |
+| wind-speed      | system.wind-speed           | R          |             |
+| wind-dir        | system.wind-direction       | R          |             |
+| pressure        | system.barometric-pressure  | R          |             |
+| visibility      | visibility-channel          | R          |             |
+| cloud-cover     | cloud-cover-channel         | R          |             |
+| solar-radiation | solar-radiation-channel     | R          |             |
+| solar-energy    | solar-energy-channel        | R          |             |
+| uv-index        | uv-index-channel            | R          |             |
+| conditions      | conditions-channel          | R          |             |
+| icon            | icon-channel                | R          |             |
+| stations        | stations-channel            | R          |             |
+| source          | source-channel              | R          |             |
+| sunrise         | sunrise-channel             | R          |             |
+| sunrise-epoch   | timestamp-channel           | R          |             |
+| sunset          | sunset-channel              | R          |             |
+| sunset-epoch    | timestamp-channel           | R          |             |
+| moon-phase      | moon-phase-channel          | R          |             |
+
+## Full Example
+
+### Thing Configuration
+
+```java
+Thing visualcrossing:weather:default_config "Total Weather Data" @ "Weather" [ apiKey="MKSH4W6U2H3BYJBB7CQVWLPTT" ]
+Thing visualcrossing:weather:all_config "Total Weather Data" @ "Weather" [ apiKey="MKSH4W6U2H3BYJBB7CQVWLPTT", location="wrocław,poland", lang="pl", hostname="https://weather.visualcrossing.com", refreshInterval="3600", httpRetries="3" ]
+```
+
+### Item Configuration
+
+Check `full_example.items` to get all days with all hours
+
+**Note: I strongly suggest not using UI Items creation because it will kill the web browser. UI is getting unresponsive when loading so many channels and trying to bind them.**
+
+**Note 2: Using `docs/full_example.items` will probably kill your OH (with message `The queue for a subscriber of type 'class org.openhab.core.internal.items.ItemUpdater' exceeds 5000 elements. System may be unstable.`). Use `docs/only_days.items` to have forecast for days but without hours.**
```

Comment:
The full_example with 10k lines is not usefull, i think it is better to remove it.

Reply:
It make sense cause you can atke parts of it to your own OH, i.e. I want hour-by-hour forecast for 3 days - then I will take only 3 days from this file and remove the rest.

If you want I can split it day by day to different files

Reply:
Good suggestion to split

Reply:
done